vuepress/docs/next/docs/releases/README.md

```diff
@@ -1,7 +1,139 @@
+
+## ABSTRACT
+
+- Security fixes
+- Node-based CLI module to manage bundle projects
+- Decoupled MFE ⇒ MS communication via service discovery ([API claims](../getting-started/ent-api.md))
+- New docker-based bundles (bundles entirely distributed via Docker/OCI images)
+- App Builder is now extensible through bundles (supports override of core App Builder components and the addition of EPC extensions)
+- App Builder main menu is converted to a overridable bundle containing an MFE and BFF service
+- Widgets, pages and bundle assets are now created within the scope of a specific bundle
+- Performance optimizations
+
+### Compatibility
+
+* [Entando 7.1 Compatibility](https://entando.com/entando-de-app/cmsresources/cms/documents/Entando_7.1_Compatibility.pdf)
+* [Guides for previous versions of Entando](https://entando.com/page/en/compatibility-guide)
+
+## BUG FIXES AND NEW FEATURES
+
+### Security Updates
+
+#### Dependencies
+
+- Fixed CVE-2022-22969 (spring-security, DoS)
+- Fixed CVE-2021-31805 (struts2, RCE)
+- Fixed CVE-2022-22965 (spring-beans, RCE)
+- Fixed CVE-2022-22950 (spring-expression, DoS)
+- Fixed CVE-2022-22968 (spring-context)
+- Fixed CVE-2022-26336 (apache poi-scratchpad, DoS)
+- Fixed CVE-2022-30126 (apache tika, DoS)
+- Fixed CVE-2022-30973 (apache tika, DoS)
+- Fixed CVE-2022-22976 (spring-security)
+- Fixed CVE-2022-22978 (spring-security-web, Authorization Bypass)
+- Fixed CVE-2022-33879 (apache tika, DoS)
+- Fixed CVE-2022-24823 (apache netty, Information Exposure)
+
+### **AdminConsole:**
+
+- Improved handling of special chars in page titles
+- Improved visual and functional integration with new AppBuilder EPC menu
+
+### App Builder
+
+- App Builder is now extensible via MFEs loaded from bundles:
+   - Provides App Builder with alternative core components
+   - Introduces new screens accessible from a menu dedicated to EPCs
+- The same SEO-friendly code can be used for different languages on the same page
+- CSS/text files can be viewed or edited in the App Builder file browser
+- Renamed Entando Component Repository (ECR) to Local Hub
+- Improved error handling when uninstalling a bundle
+- Fixed the clone function for UX fragments
+- Minor adjustments to support docker-based bundles 
+- Minor fixes related to pagination and usability
+- Invoking APIs now sends the authentication header (instead of explicitly sending authentication cookies)
+- Added a global object `window.entando` to expose global information (e.g. lang, userPermissions, router) used by MFEs to interact with App Builder
+
+### AppEngine
+
+- Enabled Keycloak authentication for additional core APIs
+- Optimized startup procedure in cases where an automatic backup restore is implied
+- All APIs now authenticate exclusively through HTTP headers and do not require cookies
+- Fixed session invalidation issues
+
+### Entando CLI (ent)
+
+- Dropped support for Entando <= 6.3.0 and Kubernetes <= 1.18
```

Comment:
Maybe I'm missing something, what does this mean? those arrows confuse things

Reply:
Less than or equal to 6.3.0, etc. Math-y.

Reply:
we can't do ≤ ?

Reply:
Hah, good idea. Correction, old-school-developer-math-y. ;)